nextflow.config

```diff
@@ -231,13 +231,13 @@ dag {
 }
 
 manifest {
-    name            = 'nf-core/staramr'
-    author          = """Eric Marinier"""
-    homePage        = 'https://github.com/nf-core/staramr'
+    name            = 'phac-nml/staramrnf'
+    author          = """Steven Sutcliffe and Eric Marinier"""
+    homePage        = 'https://github.com/phac-nml/staramrnf'
     description     = """StarAMR: Scans genome contigs against the ResFinder, PlasmidFinder, and PointFinder databases."""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version         = '1.0dev'
+    version         = '0.1.0'
```

Comment:
Could you also add `defaultBranch = 'main'` here, like in the speciesabundance repo: https://github.com/phac-nml/speciesabundance/blob/main/nextflow.config#L213

Sorry for not mentioning earlier. I just remembered this.

Reply:
Added bb623e8